views/fieldinput.erb

```diff
@@ -0,0 +1,57 @@
+<% def even?(n)
+  if n % 2 == 0
+    even = true
+  else
+    even = false
+  end
+end %>
+
+<h3>Edit Profile</h3>
+
+<form action="/update" method="POST">
+  <fieldset>
+    <table>
+      <tbody>
+        <tr>
+          <td>
+            <div class="profile-section box">
+                <label for="trainee[about]">About</label>
+                <br>
+                <textarea rows="3" cols="40" name="trainee[about]" value="<%= @personal_info["about"] %>"></textarea>
```

Comment:
With textarea, the value goes in between the textarea tags as a text node. Try it and it should show up